.circleci/config.yml

```diff
@@ -61,6 +61,47 @@ jobs:
           command: |
             cd build
             make install
+  build_gpu:
+    machine:
+      resource_class: gpu.nvidia.small
+      image: ubuntu-1604-cuda-10.1:201909-23
```

Comment:
What images are available? I very much suspect we need to test with multiple versions of CUDA, at least the ones supported/tested by PyTorch.

Reply:
There's only ubuntu 16.04 with either cuda 10.1 or 9.2 available. If we want to test against multiple versions of cuda, the easiest way will be to build docker containers. The scope of this PR is simply to get _some_ signal (i.e. ensure it builds fine and it is not broken in an obvious way), but we can discuss about more thorough coverage.